homeassistant/components/sensor/flunearyou.py

```diff
@@ -0,0 +1,212 @@
+"""
+Support for user- and CDC-based flu info sensors from Flu Near You.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.flunearyou/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, ATTR_STATE, CONF_LATITUDE, CONF_MONITORED_CONDITIONS,
+    CONF_LONGITUDE)
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pyflunearyou==0.0.2']
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_CITY = 'city'
+ATTR_REPORTED_DATE = 'reported_date'
+ATTR_REPORTED_LATITUDE = 'reported_latitude'
+ATTR_REPORTED_LONGITUDE = 'reported_longitude'
+ATTR_ZIP_CODE = 'zip_code'
+
+DEFAULT_ATTRIBUTION = 'Data provided by Flu Near You'
+DEFAULT_SCAN_INTERVAL = timedelta(minutes=30)
```

Comment:
This makes your sensors update every 30 seconds (the `sensor` default) but new data will only be available every 30 minutes.

Consider naming this `MIN_TIME_BETWEEN_UPDATES` instead and lower it a bit.

Then set `SCAN_INTERVAL` to 30 minutes so Home Assistant actually only updates the sensor every 30 minutes.